fpga_interchange/interchange_capnp.py

```diff
@@ -243,8 +243,9 @@ def create_property_map(self, property_map, d):
             entry.key = self.string_id(k)
 
             if isinstance(v, str):
-                if v[0] == '"' and v[-1] == '"':
-                    v = v[1:-1]
+                if len(v) >= 2:
```

Comment:
Can you add some unit tests around this code?

Reply:
Might be best to refactor the code first, and then write some tests on the separated function.

Reply:
This PR still needs some tests?